modules/github-azure-oidc/variables.tf

```diff
@@ -0,0 +1,92 @@
+#Resource Group Variables
+variable "rg_create" {
+  description = "Create resource group. When set to false, uses id to reference an existing resource group."
+  type        = bool
+  default     = true
+}
+
+variable "rg_name" {
+  type        = string
+  description = "The name of the resource group to create the github foundation azure resources in."
+}
+
+variable "rg_location" {
+  type        = string
+  description = "The location of the resource group to create the github foundation azure resources in."
+}
+
+#Storage Variables
+variable "sa_name" {
+  type        = string
+  description = "The name of the storage account for github foundations."
+}
+
+variable "sa_tier" {
+  type        = string
+  description = "The tier of the storage account for github foundations. Valid options are Standard and Premium."
+}
+
+variable "sa_replication_type" {
+  type        = string
+  description = "The replication type of the storage account for github foundations. Valid options are LRS, GRS, RAGRS, ZRS, GZRS, and RA_GZRS."
+}
```

Comment:
Should we select a sensible default value here?

Reply:
Set to Standard and GRS. The defaults you get in the az portal